src/main/scala/mesosphere/marathon/storage/StorageConfig.scala

```diff
@@ -240,6 +245,7 @@ object StorageConfig {
 
   val DefaultLegacyMaxVersions = 25
   val DefaultMaxVersions = 5000
+  val DefaultScanBatchSize = 32
```

Comment:
🤔 isn't the Scallop option already defining a default?

Reply:
well, it's not for the scallop, but rather for typesafe config which is already here. You can check the line 236

Reply:
So why not pull in the default from the Lightbend config?

Reply:
This whole method is never used. I guess at some point someone wanted to switch to `typesafe.Config` eventually? I'd rather remove it completely in a separate PR. It's dead code anyway.